Cargo.toml

```diff
@@ -12,8 +12,10 @@ serde = { features = ["serde_derive"], optional = true, version = "1" }
 arbitrary = { features = ["derive"], optional = true, version = "1" }
 
 [features]
+default = ["std"]
+std = []
 serialize = ["serde"]
```

Comment:
I'm wondering if users will want to use `serde` in combination with `no-std`? :thinking: And it seems like there is support https://serde.rs/no-std.html, but if they enabled `serialize` without `std` I think `std` would then be enabled via the `serde`

Reply:
Hmmm... I tried adding `default-features = false` to the `serde` dependency but then `cargo check --target thumbv7em-none-eabihf --no-default-features --features serialize` fails for me.

Reply:
Fixed.